app/screens/Table/components/Species.js

```diff
@@ -146,9 +146,9 @@ class Species extends React.Component {
       let extraPokemonNeededSpan
 
       if (specie.evolves > 0) {
-        const totalCandyNeeded = specie.candyToEvolve * specie.count
+        const totalCandyNeeded = (specie.candyToEvolve - 1) * specie.count + 1
         const extraCandyNeeded = totalCandyNeeded - specie.candy
-        const extraPokemonNeeded = Math.floor((extraCandyNeeded * -1) / specie.candyToEvolve)
+        const extraPokemonNeeded = Math.floor((extraCandyNeeded * -1) / (specie.candyToEvolve - 1))
 
```

Comment:
I don't know where this `((extraCandyNeeded * -1) ` comes from, but the important change here would be to divide a pidgey's needed candies by 11, not 12. 

`(specie.candyToEvolve - 1))` would be correct, unless there's just the 1 pidgey to evolve.